panel/tests/chat/test_langchain.py

```diff
@@ -21,8 +21,8 @@ async def callback(contents, user, instance):
     tools = load_tools(["python_repl"])
     responses = ["Action: Python REPL\nAction Input: print(2 + 2)", "Final Answer: 4"]
     llm_kwargs = dict(responses=responses, callbacks=[callback_handler], streaming=streaming)
-    if streaming and Version(platform.python_version()) <= Version("3.8"):
-        pytest.skip("Streaming on FakeListLLM not supported on Python 3.8")
+    if Version(platform.python_version()) <= Version("3.8"):
```

Comment:
You could use `sys.version_info < (3, 8)` here.

But we don't support 3.8 anymore. So I don't think this is needed.